README.md

```diff
@@ -6,11 +6,15 @@
 
 # SPDX tools-golang
 
-tools-golang is a collection of Go packages intended to make it easier for
+`tools-golang` is a collection of Go packages intended to make it easier for
 Go programs to work with [SPDX®](https://spdx.dev/) files.
 
 ## Recent news
 
+2023-01: **v0.5.0**: refactored the data model to be consistent for consumers
```

Comment:
Added this version based on the roadmap here: https://github.com/spdx/tools-golang/issues/155